game/static/game/css/board.css

```
@@ -318,6 +359,22 @@ body {
+    --layout-padding-x: clamp(10px, 1.5vw, 20px);
+    --layout-padding-y: clamp(12px, 1.5vw, 20px);
+    --panel-width: clamp(128px, 11vw, 196px);
+    --rank-label-width: 1.35rem;
+    --board-border: 3px;
+    --board-chrome: calc(var(--rank-label-width) + var(--board-border) * 2 + 12px);
+    /* Fit board + 3 side panels within the viewport */
+    --square-size: clamp(
+        26px,
+        calc(
+            (100vw - 3 * var(--panel-width) - 3 * var(--layout-gap)
+                - 2 * var(--layout-padding-x) - var(--board-chrome)) / 8
```

Stylelint (17.11.1)

[error] Unexpected newline before "-" (scss/operator-no-newline-before)

(scss/operator-no-newline-before)